lib/heborn_migration/model/account.ex

```diff
@@ -69,6 +69,19 @@ defmodule HEBornMigration.Model.Account do
     NaiveDateTime.diff(now, struct.inserted_at) > @expiration_time
   end
 
+  @spec format_error(Ecto.Changeset.t) ::
+    %{atom => [String.t]}
+  @doc """
+  Formats changeset errors
+  """
+  def format_error(changeset) do
+    Ecto.Changeset.traverse_errors(changeset, fn {msg, opts} ->
+      Enum.reduce(opts, msg, fn {key, value}, acc ->
+        String.replace(acc, "%{#{key}}", to_string(value))
+      end)
+    end)
+  end
```

Comment:
Maybe there's a better way for doing that, I remember (reading) that phoenix having something better for doing this kind of thing, but I can't find it anywhere.